bor/beginblocker.go

```diff
@@ -0,0 +1,41 @@
+package bor
+
+import (
+	"encoding/json"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/maticnetwork/bor/consensus/bor"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/maticnetwork/heimdall/bor/client/rest"
+	"github.com/maticnetwork/heimdall/helper"
+	hmTypes "github.com/maticnetwork/heimdall/types"
+)
+
+func BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock, k Keeper) {
+
+	if ctx.BlockHeight() == int64(helper.SpanOverrideBlockHeight) {
+		k.Logger(ctx).Info("overriding span BeginBlocker", "height", ctx.BlockHeight())
+		j, ok := rest.SPAN_OVERRIDES[helper.GenesisDoc.ChainID]
+		if !ok {
+			k.Logger(ctx).Error("Error in fetching span overrides")
```

Comment:
return statement

Reply:
should we panic instead? This error is not intended right?

Reply:
error is not intended
i was thinking of return instead of panic

Reply:
paniced